src/quacc/calculators/espresso/presets/bulk_metal_efficiency.yaml

```diff
@@ -0,0 +1,635 @@
+# SSSP PBE Efficiency v1.3.0
+# https://doi.org/10.1038/s41524-018-0127-2
+# https://www.materialscloud.org/discover/sssp/table/efficiency
+
+input_data:
+  system:
+    occupations: smearing
+    smearing: marzari-vanderbilt
+    degauss: 0.01
+  electrons:
+    conv_thr: 1.0e-10
+    mixing_mode: TF
+    mixing_beta: 0.7
+
+kspacing: 0.25
+
+pseudopotentials:
```

Comment:
Can probably be `parent_pseudopotentials: sssp_1.3.0_pbe_efficiency.yaml` if I understand correctly.